uint8[78] data